react/week3/todo/src/components/TodoForm.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+function TodoForm({ todos, setTodos }) {
+  const [inputTodo, setInputTodo] = useState("");
+  const [inputDate, setInputDate] = useState("");
+
+  const buttonAddTodo = () => {
```

Comment:
This is a function, and the naming needs to be only descriptive of what it does ie. adding a todo. So `addTodo` would suffice.